deployments/puppet/manifests/collector_win_config_options.pp

```diff
@@ -27,5 +27,5 @@
             {}
         }
 
-    $collector_env_vars = stdlib::merge($base_env_vars, $gomemlimit, $listen_interface)
-}
\ No newline at end of file
+    $collector_env_vars = $base_env_vars + $gomemlimit + $listen_interface
```

Comment:
Per documentation this is the same as stdlib::merge, see https://forge.puppet.com/modules/puppetlabs/stdlib/reference#stdlibmerge